src/libraries/System.Diagnostics.DiagnosticSource/tests/DiagnosticSourceTests.cs

```diff
@@ -506,6 +506,7 @@ public void AllListenersCheckCatchupList()
         [InlineData(100, 102)]
         [InlineData(100, 103)]
         [InlineData(100, 104)]
+        [PlatformSpecific(~TestPlatforms.Browser)] // Cannot wait on monitors on this runtime.
```

Comment:
Same `PlatformDetection.IsThreadingSupported` change here, and other tests which are disabled because of monitors.

Reply:
Addressed in recent commits